stl/inc/atomic

```diff
@@ -2041,6 +2295,32 @@ struct atomic_flag { // flag with test-and-set semantics
 
     constexpr atomic_flag() noexcept = default;
 
+#if _HAS_CXX20
+    void wait(const bool _Expected, memory_order _Order = memory_order::seq_cst) const noexcept {

```

Comment:
I observe that we're accumulating inconsistency - old code says `memory_order_seq_cst` but C++20 code is saying `memory_order::seq_cst`. No change requested because it's pre-existing.

Reply:
I think it was copied from the Standard. [It was memory_order_seq_cst, now it is memory_order::seq_cst](https://en.cppreference.com/w/cpp/atomic/memory_order)

Reply:
We can't unconditionally do that so I'll make them all `memory_order_seq_cst`.